<launch>

    <!--Load parameters-->
    <rosparam command="load" file="$(find iiwa_command)/config/params_fri_position.yaml"/>
    <rosparam command="load" file="$(find iiwa_command)/config/iiwa_robot_parameters.yaml"/>

    <!-- Launch iiwa_command nodes -->
    <node pkg="iiwa_command" name="iiwa_command_fri" type="iiwa_command_fri" respawn="true" output="screen" launch-prefix="xterm -e"/>
    <node pkg="iiwa_command" name="msg_transform_helper" type="msg_transform_helper" respawn="true" output="screen" launch-prefix="xterm -e"/>
</launch>
